backend/app/api/endpoints/adapter/teams.py

```
@@ -124,4 +126,92 @@ def join_shared_team(
+def get_recommended_teams(
+    limit: int = Query(6, ge=1, le=20, description="Max teams to return"),
+    db: Session = Depends(get_db),
+    current_user: User = Depends(security.get_current_user)
+):
+    """Get recommended teams (is_recommended=true)"""
+    from app.schemas.kind import Team
+
+    # Get all teams where isRecommended is true
+    teams = db.query(Kind).filter(
+        Kind.kind == "Team",
+        Kind.is_active == True
```

Ruff (0.14.6)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)